.github/workflows/cron-test.yml

```diff
@@ -42,3 +42,7 @@ jobs:
           S3_BUCKET_NAME: ${{ secrets.S3_BUCKET_NAME }}
         run: |
           ./scripts/run-integration-tests.sh
+
+      - name: Run new integration tests
+        run: |
+          ./scripts/test/run-integration-tests.sh 
```

Comment:
Will the script work without any environment variables?
Also, I prefer doing this as part of a different PR so it's easy to revert if there are issues.

Reply:
@Shreya027 - You can test this on your local fork. Lets add the new script/test cases for nightly runs since it has been long pending.